### PR TITLE
disclaimers

### PR DESCRIPTION
**Pull request by ebuchman**

### CODE REVIEW

**Comment by leoluk** [comment]
Oops. You broke it :joy:

File: README.md [hunk 1]
```diff
@@ -46,6 +56,11 @@ all components of the genesis file and scripts to recompute it.
 Anyone with an ATOM allocation in the [`penultimate_genesis.json`](./penultimate_genesis.json) who intends to participate in the genesis ceremony must submit a pull request
 containing a valid `gen-tx` to this repository in the `/gentx` folder with a file name like `<moniker>.json`.
 
+It must be noted that it remains in the community's discretion to adopt or not
+to adopt the Genesis State that the ICF recommends within the Genesis Block
+Software. Therefore, the ICF cannot guarantee that (i) ATOMs will be created and
+(ii) the recommended allocation as set forth herein will actually take place
+
```
Comment (nylira) [inline README.md]: This seems like a repeat of the following section in the Disclaimer?

**Comment by leoluk** [comment]
You broke it! :joy: 

The commit had a date in the future, which sent the bot into a loop. Never trust user input... :joy:

File: README.md (hunk 1 above)
Comment (ebuchman) [reply README.md]: Purposeful repetition :)

Comment (ebuchman) [reply README.md]: Ok I eased it.

**Comment by ebuchman** [comment]
> The commit had a date in the future, which sent the bot into a loop. Never trust user input... :-)

Lol amazing

**Pull request merged**